ai/gpu docker/Readme.txt

```diff
@@ -0,0 +1 @@
+The Dockerfile works for both GPU training and prediction. Please build it with the docker version of Nvidia-docker2.
```

Comment:
Please add detailed commands to build and run container for both training and prediction.